front/lib/api/assistant/agent.ts

```diff
@@ -191,12 +193,22 @@ export async function* runAgent(
   | DatabaseQueryParamsEvent,
   void
 > {
+  const detailedConfiguration = await getAgentConfigurationDetailedView(
```

Comment:
Same here let's find a name and commit.

If we go with detailedConfiguration then it should be detailedConfiguration everywhere (no detailConfig) :)

Reply:
Still not a big fan of detailedConfiguration because this is the configuration really.

I see the value of using configuration for the light version because it limits the number of variable name changes.

Maybe `configuration` (can be used for both) and `lightConfiguration` or `fullConfiguration` (when disambiguation is needed in one way or another) ?

Reply:
Agreed will update